src/main/groovy/org/checkerframework/gradle/plugin/CheckerFrameworkPlugin.groovy

```diff
@@ -304,12 +309,13 @@ final class CheckerFrameworkPlugin implements Plugin<Project> {
         createManifestTask.checkers = userConfig.checkers
       }
 
-      project.tasks.withType(AbstractCompile).all { compile ->
-        if (compile.hasProperty('options') && !(userConfig.excludeTests && compile.name.toLowerCase().contains("test"))) {
-          compile.options.annotationProcessorPath =
-                  compile.options.annotationProcessorPath == null ?
-                          project.configurations.checkerFramework :
-                          project.configurations.checkerFramework.plus(compile.options.annotationProcessorPath)
+      project.tasks.withType(AbstractCompile).all { AbstractCompile compile ->
```

Comment:
Could you please use `configureEach` instead of `all`?
`all` results in eager task realization, while `configureEach` supports [configuration avoidance](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html)

Reply:
Thanks for catching this! Resolved in #115.